src/network/core_utils.rs

```diff
@@ -25,6 +25,129 @@ pub struct CoreUtils {
     pub networkns: String,
 }
 
+pub struct IPAMAddresses {
```

Comment:
This can be moved to https://github.com/containers/netavark/blob/main/src/network/internal_types.rs

Reply:
yeah good idea